libraries/wasm-jit/Source/Logging/Logging.cpp

```diff
@@ -10,11 +10,7 @@ namespace Log
 	static bool categoryEnabled[(Uptr)Category::num] =
 	{
 		true, // error
-		#ifdef _DEBUG // debug
-			true,
-		#else
-			false,
-		#endif
+	  false, // debug
```

Comment:
wasm-jit has a different indentation style

Reply:
Thanks. Fixed. It was caused by a tab (wasm-jit uses tabs for indentation).